### DISCUSSION TITLE
"REST API error 400: model_error" when using the TypeChat samples

### DISCUSSION THREAD

**Discussion by angusforeman** [microsoft/TypeChat#63] [Q&A]
Anyone else experiencing this error when using the TypeChat samples. Happens for Sentiment & Math examples (and I assume all samples)
Environment: Running Node 18.17 for Windows, using the Azure OpenAI models, the .env files are correctly formatted (copied from OpenAI playground). 

Thanks

**Comment by DanielRosenwasser** [comment]
Not entirely sure - closest thing I've found to this is [this StackOverflow question](https://stackoverflow.com/a/75884736).

Is there a way you can share an anonymized version of your endpoint URL?

**Reply by gbangd** [reply]
I'm experiencing the same issue. This is my environment variable: AZURE_OPENAI_ENDPOINT=https://<myresource>.openai.azure.com/openai/deployments/<mydeployment>/completions?api-version=2022-12-01

**Reply by gbangd** [reply]
I've found the issue. I was getting the endpoint url and api key from the Completions Playground in Azure and not from the Chat Playground. After getting it from the Chat Playground, examples started to work properly.
![image](https://github.com/microsoft/TypeChat/assets/1699709/b0f281c4-9eb0-454d-8ff2-b00cc1793991)
